From 9a52374f4609fe42ed46332bf72cb0b9a6c868ba Mon Sep 17 00:00:00 2001
From: Harman Kalra <hkalra@marvell.com>
Date: Mon, 3 Feb 2020 19:28:35 +0530
Subject: [PATCH 468/767] net: thunder: renaming thunder nicpf and vf module
 name

commit 7dea0a0ea48cb31cf4f1eb370aa64a5b983f25fc from
git@git.assembla.com:cavium/WindRiver.linux.git

There is a mismatch in driver name and module name for
thunder pf and vf driver in sysfs entries.
 ls -l /sys/bus/pci/drivers/thunder-nicvf/module
/sys/bus/pci/drivers/thunder-nicvf/module -> ../../../../module/nicvf
                     ............                              ------

 ls -l /sys/module/nicvf
                  ......

In general they should be same else it might lead to some problems
where comaprision between both is made to perform a task.

Change-Id: I8ea3913072db3818bc156206aba62bc5a9c05a26
Signed-off-by: Harman Kalra <hkalra@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/22834
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/cavium/thunder/Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/cavium/thunder/Makefile b/drivers/net/ethernet/cavium/thunder/Makefile
index 2fc6142d1634..6cf739e31883 100644
--- a/drivers/net/ethernet/cavium/thunder/Makefile
+++ b/drivers/net/ethernet/cavium/thunder/Makefile
@@ -5,9 +5,9 @@
 
 obj-$(CONFIG_THUNDER_NIC_RGX) += thunder_xcv.o
 obj-$(CONFIG_THUNDER_NIC_BGX) += thunder_bgx.o
-obj-$(CONFIG_THUNDER_NIC_PF) += nicpf.o
-obj-$(CONFIG_THUNDER_NIC_VF) += nicvf.o
+obj-$(CONFIG_THUNDER_NIC_PF) += thunder_nic.o
+obj-$(CONFIG_THUNDER_NIC_VF) += thunder_nicvf.o
 
-nicpf-y := nic_main.o
-nicvf-y := nicvf_main.o nicvf_queues.o
-nicvf-y += nicvf_ethtool.o
+thunder_nic-y := nic_main.o
+thunder_nicvf-y := nicvf_main.o nicvf_queues.o
+thunder_nicvf-y += nicvf_ethtool.o
-- 
2.31.1

